velox/type/Filter.h

```diff
@@ -80,6 +81,12 @@ class Filter : public velox::ISerializable {
   // runtime.
   static constexpr bool deterministic = true;
 
+  virtual bool testBloomFilter(
+      const AbstractBloomFilter& bloomFilter,
+      const velox::Type& type) const {
```

Comment:
No need to pass `type` as integers should give same results no matter how wide it is.

Reply:
Please see the comment above. We need to do type specific checks.